### ISSUE TITLE
PGML numeric lists should work with )

### ISSUE THREAD

**Issue by somiaj** [openwebwork/pg#748]
When creating a numeric list with PGML, the following doesn't work:

```
1) First
2) Second
3) Third
```

This does work with all other lists types I've tests, `a)`, `A)`, `i)`, and `I)`. So in order to create a numeric list `1.` is needed instead. To me this is inconsistent. Though the docs seem to only state `)` is usable with alpha characters (<https://webwork.maa.org/wiki/Lists_and_sub-lists_-_PGML>).

I think it would be useful if the above would create a list to be consistent with other lists. This one case being different at least caused me confusion for a bit.

**Comment by dpvc** [comment]
It looks like the problem is 

https://github.com/openwebwork/pg/blob/27923e4770dd64bfa76b8dd15d0d8a9e8f0f4bdb/macros/PGML.pl#L155

which should be

``` perl
    /\d+[.)] /       && do {return $self->Bullet($token,"numeric")};
```

like the lines below it.  It might also be good to change

https://github.com/openwebwork/pg/blob/27923e4770dd64bfa76b8dd15d0d8a9e8f0f4bdb/macros/PGML.pl#L40

to be

``` perl
my $list = '(?:^|(?<=[\t ]))(?:[-+o*]|(?:\d+|[ivxl]+|[IVXL]+|[a-zA-Z])[.)]) +';
```

to allow multi-digit list numbers as input.

**Comment by drgrice1** [comment]
This is fixed by #750.  Your own pull request that has been merged!
